tests/unit/backend/gene_info/test_v1.py

```diff
@@ -0,0 +1,127 @@
+import unittest
+import requests
+import json
+
+from backend.gene_info.api import ncbi_provider
+from tests.unit.backend.corpora.fixtures.environment_setup import EnvironmentSetup
+from backend.corpora.api_server.app import app
+import xml.etree.ElementTree as ET
+from unittest.mock import patch, call
+
+
+class GeneInfoAPIv1Tests(unittest.TestCase):
+    """Tests Gene Info API endpoint"""
+
+    def setUp(self):
+        super().setUp()
+        with EnvironmentSetup(dict(APP_NAME="corpora-api")):
+            self.app = app.test_client(use_cookies=False)
+        self.final_gene_info_result = {
+            "name": "",
+            "summary": "",
+            "ncbi_url": "https://www.ncbi.nlm.nih.gov/gene/1",
+            "synonyms": [],
+        }
+
+    @classmethod
+    def setUpClass(cls) -> None:
+        super().setUpClass()
+        cls.maxDiff = None
+
+    @patch("backend.gene_info.api.ncbi_provider.urllib.request.urlopen")
+    @patch("backend.gene_info.api.v1.NCBIProvider._get_api_key")
+    @patch("backend.gene_info.api.v1.NCBIProvider._load_search_result")
+    def test_api_calls(self, mock_load_search_result, mock_api_key, mock_get):
+        """
+        Mocks API key for NCBI requests, checks call counts and the correct external API calls
+        Will break if the external API call is down!
+        """
+        mock_get.read = None
+        mock_load_search_result.return_value = 348
+        mock_api_key.return_value = ""
+        test_search_url = (
+            "https://eutils.ncbi.nlm.nih.gov/entrez/eutils/esearch.fcgi?" "db=gene&term=ENSG00000130203&retmode=json"
+        )
+        test_fetch_url = "https://eutils.ncbi.nlm.nih.gov/entrez/eutils/efetch.fcgi?db=gene&id=348&retmode=xml"
+        test_provider = ncbi_provider.NCBIProvider()
+        test_provider._get_api_key.return_value = ""
+        test_provider.fetch_gene_uid("ENSG00000130203")
+        test_provider.fetch_gene_info_tree(348)
+        mock_get.assert_has_calls([call(test_search_url), call().read(), call(test_fetch_url), call().read()])
+        self.assertEqual(mock_get.call_count, 2)
+
+    @patch("backend.gene_info.api.v1.NCBIProvider.fetch_gene_uid")
+    @patch("backend.gene_info.api.v1.NCBIProvider.fetch_gene_info_tree")
+    @patch("backend.gene_info.api.v1.NCBIProvider.parse_gene_info_tree")
+    def test_fetches_and_searches(self, mock_parse_gene_info_tree, mock_fetch_gene_info_tree, mock_fetch_gene_uid):
+        """
+        Successfully calls NCBIProvider fetch and search functions with correct parameters
+        """
+        mock_fetch_gene_uid.return_value = 1
+        mock_fetch_gene_info_tree.return_value = None
+        mock_parse_gene_info_tree.return_value = self.final_gene_info_result
+        res = self.app.get("/gene_info/v1/gene_info?geneID=ensembl1")
+        assert mock_fetch_gene_uid.called is True
```

Comment:
Use the `unittest` test assertions
```suggestion
        self.assertTrue(mock_fetch_gene_uid.called)
```